pkg/scheduler/objects/preemption.go

```diff
@@ -174,6 +177,20 @@ func (p *Preemptor) initWorkingState() {
 	p.allocationsByNode = allocationsByNode
 	p.queueByAlloc = queueByAlloc
 	p.nodeAvailableMap = nodeAvailableMap
+	p.nodeInstanceTypeMap = nodeInstanceTypeMap
+}
+
+// getInstanceType retrieves instance type from the nodeInstanceTypeMap for given nodeId
+// if not found, issue error log and return UnknownInstanceType
+func (p *Preemptor) getInstanceType(nodeId string) string {
```

Comment:
This doesn't really need its own method.

Reply:
Very much appreciated @craigcondit uploaded new rev again to address both latest points you made. Thanks